datashare-app/src/test/java/org/icij/datashare/web/RootResourcePluginTest.java

```diff
@@ -67,12 +68,13 @@ public void test_get_with_no_plugin_directory() {
     }
 
     @Test
-    public void test_invalid_folder_should_throw_error() {
-        server.configure(routes -> routes.add(new RootResource(new PropertiesProvider(new HashMap<String, String>() {{
-            put("pluginsDir", "unknown");
-        }}))));
-        get("/").should().respond(500);
+    public void test_get_with_missing_plugin_directory() {
+        propertiesProvider = new PropertiesProvider(Map.of("pluginsDir", "/something/missing/"));
+        server.configure(routes -> routes.add(new RootResource(propertiesProvider)));
+        get("/").should().respond(200).contain("datashare-client");
+        get("").should().respond(200).contain("datashare-client");
     }
+    
```

Comment:
The unit test `test_invalid_folder_should_throw_error` has been removed. Is it not relevant anymore?

Reply:
Nope because it's not throwing an error anymore, it logs a warnning :)  Thanks for the review!